components/bl-mention-component/styles/primereact.less

```diff
@@ -0,0 +1 @@
+.p-component,.p-component *{box-sizing:border-box}.p-hidden{display:none}.p-hidden-space{visibility:hidden}.p-hidden-accessible{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.p-hidden-accessible input,.p-hidden-accessible select{transform:scale(0)}.p-disabled,.p-disabled *{cursor:default!important;pointer-events:none;user-select:none}.p-component-overlay{position:fixed;top:0;left:0;width:100%;height:100%}.p-connected-overlay{opacity:0;transform:scaleY(.8);transition:transform .12s cubic-bezier(0, 0, .2, 1),opacity .12s cubic-bezier(0, 0, .2, 1)}.p-connected-overlay-visible{opacity:1;transform:scaleY(1)}.p-connected-overlay-hidden{opacity:0;transform:scaleY(1);transition:opacity .1s linear}.p-connected-overlay-enter{opacity:0;transform:scaleY(.8)}.p-connected-overlay-enter-active{opacity:1;transform:scaleY(1);transition:transform .12s cubic-bezier(0, 0, .2, 1),opacity .12s cubic-bezier(0, 0, .2, 1)}.p-connected-overlay-enter-done{transform:none}.p-connected-overlay-exit{opacity:1}.p-connected-overlay-exit-active{opacity:0;transition:opacity .1s linear}.p-inputtext{margin:0}.p-inputtextarea-resizable{overflow:hidden;resize:none}.p-mention{display:inline-flex;position:relative}.p-mention-panel{position:absolute;top:0;left:0;overflow:auto}.p-mention-items{margin:0;padding:0;list-style-type:none}.p-mention-item{cursor:pointer;white-space:nowrap;position:relative;overflow:hidden}.p-fluid .p-mention{display:flex}
```

Comment:
For a case, if it not rendered in the root - please wrap it in our class.
Or if component has api for adding class to this popup - do it.